src/helpers.ts

```diff
@@ -36,7 +36,10 @@ export function readSettings<T>(serviceName: string, settingsFileName: string):
   if (parseErrors && parseErrors.length > 0) {
     throw new Error(
       `[${serviceName}] Unable to load configuration file: ${parseErrors
-        .map(error => log.error("${serviceName}", `${error?.error}`))
+        .map(error => {
+            log.error(`${serviceName}`, `${error?.error}`);
+            return JSON.stringify(error);
+        })
```

Comment:
Because the anonymous function was returning nothing, `join` had nothing to join on.